src/Components/SearchResults.tsx

```diff
@@ -10,22 +10,55 @@ import { serialize } from 'v8';
 
 const SearchResults = () => {
 
-  const { events, searchedEvents, category, org, filteredEvents, filterByCategory, setEvents, setSearchedEvents } = useContext(EventContext)
+  const { setCategory, category, org } = useContext(EventContext)
+  const [events, setEvents] = useState([])
+  const [filteredEvents, setFilteredEvents] = useState<any>(events)
   const zip = useParams().zipcode
   const miles = useParams().mileage
+
   console.log(zip)
   console.log(miles)
-  console.log('searchresults', searchedEvents)
+  
 
   useEffect(() => {
     apiCalls.loadEventsByZipCode(zip, miles)
-      .then(data => setSearchedEvents(data.data))
+      .then(data => setEvents(data.data))
+      .then(events => setFilteredEvents(events))
```

Comment:
I'm not sure if this part actually works the way I want it to. Right now, the code here functions as we want, but I am not sure if line 26 here actually does anything when this component loads.